packages/base/src/client.ts

```diff
@@ -390,8 +390,8 @@ export class AppKit<AdapterStoreState = unknown, SwitchNetworkParam = unknown> {
     OptionsController.setPrivacyPolicyUrl(options.privacyPolicyUrl)
     OptionsController.setCustomWallets(options.customWallets)
     OptionsController.setEnableAnalytics(options.enableAnalytics)
-    OptionsController.setOnrampEnabled(options.enableOnramp !== false)
-    OptionsController.setEnableSwaps(options.enableSwaps !== false)
+    OptionsController.setOnrampEnabled(options.enableOnramp)
```

Comment:
@zoruka Onramp and Swaps will be enabled by default. So when they don't provide these props, they'll be `undefined`. To be able to set them `true`, we need `!== false` options to enable them

Reply:
I did add the default parameter in the [constant file](https://github.com/reown-com/appkit/pull/2935/files#diff-5db4770c881137b47d85a901a5f7b972e5845d481e09084aa31cdd653cdce0a4R205-R206) so we don't need hardcoded values like that. Are you sure this is correct?